examples/layers/ios/.gitignore

```diff
@@ -0,0 +1,37 @@
+.idea/
```

Comment:
LGTM for the Pods stuff. We've traditionally checked in everything for Pods, so it's possible to build the project without having CocoaPods installed. Otherwise, we'd have Pods/ and Podfile.lock in .gitignore.